Source/SwiftLintBuiltInRules/Rules/Performance/EmptyCountRule.swift

```diff
@@ -137,6 +153,14 @@ private extension TokenSyntax {
     }
 }
 
+private extension MacroExpansionExprSyntax {
+    var isTipsRuleMacro: Bool {
+        self.macroName.text == "Rule" &&
+        self.arguments.isNotEmpty &&
+        self.trailingClosure != nil
```

Comment:
Let's be even more restrictive here:
```suggestion
        macroName.text == "Rule" &&
        arguments.count == 1 &&
        trailingClosure.map { $0.statements.onlyElement?.item.is(ReturnStmtSyntax.self) == false } ?? false
```